applications/fluid_structure_interaction/perpendicular_flap/application.h

```diff
@@ -151,12 +151,13 @@ class Application : public FluidFSI::ApplicationBase<dim, Number>
     param.treatment_of_convective_term    = TreatmentOfConvectiveTerm::Explicit;
     param.order_time_integrator           = 1;
     param.start_with_low_order            = true;
-    param.adaptive_time_stepping          = false;
-    param.calculation_of_time_step_size   = TimeStepCalculation::UserSpecified;
+    param.adaptive_time_stepping          = true;
+    param.calculation_of_time_step_size   = TimeStepCalculation::CFL;
     param.time_step_size                  = DELTA_T;
+    param.time_step_size_max              = DELTA_T;
     param.max_velocity                    = U_MEAN;
-    param.cfl                             = 0.5;
-    param.cfl_exponent_fe_degree_velocity = 1.5;
+    param.cfl                             = 0.75;
+    param.cfl_exponent_fe_degree_velocity = 1.2;
```

Comment:
Out of curiosity, where does this change (1.5 -> 1.2) come from? Did you make parameter studies for different polynomial degrees?

Reply:
Short answer: no. Before the current changes, I used a fixed time-step size so that this parameters had no influence on the final result. I tried (and ran) the setup now with adaptive time-stepping, but the computed time-step size is very small (compared to the fixed one). The perpendicular flap case is somewhat artificial in the sense that the setup has a very high viscosity (low Re) so that bigger time-step sizes are acceptable and do not lead to unstable behavior. Hence, I just tried to loosen up the time-step restrictions.

Reply:
Then, I would use the standard value 1.5 for the exponent and adjust the time step size via larger cfl values.